docs/source/json-api/production-setup.rst

```diff
@@ -4,18 +4,10 @@
 Production Setup
 ################
 
-The vast majority of the prior documentation focuses on ease of testing and running
-the service in a dev environment. From a production perspective given the wide
-variety of use-cases there is far less of an established framework for deploying
-the *HTTP JSON API* server. In this document we would try to list some recommendations for
-production deployments.
+The vast majority of prior documentation focused on ease of testing and on setting up the service to run in a dev environment. From a production perspective, given the wide variety of use-cases, there is far less of an established framework for the deployment of an *HTTP JSON API* server. In this document we will make some recommendations for production deployments.
```

Comment:
```suggestion
The majority of prior documentation focused on ease of testing and on setting up the service in a dev environment. From a production perspective, given the wide variety of use cases, there is far less of an established framework for the deployment of an *HTTP JSON API* server. In this document we will make some recommendations for production deployments.
```

Reply:
I'm not sure we should remove "to run" 

removing "vast" and changing it to use case are good tho

Reply:
@ShimonZerbib In general, please keep the newlines as is to avoid needless noise that makes the review more difficult. Those are effectively ignored by the documentation generator so there's not real reason to change them.

Reply:
I'm not married to the idea of removing "to run", I always lean towards using the minimum number of words to express the idea but this is a style issue and not a right/wrong thing.

Reply:
I understand. If you're ok I'll keep the initial edit (admittedly a stylistic one). I do think "use-case" should be hyphenated though.